sklearn/metrics/pairwise.py

```diff
@@ -1215,7 +1215,7 @@ def kernel_metrics():
 
 KERNEL_PARAMS = {
     "additive_chi2": (),
-    "chi2": (),
+    "chi2": frozenset(["gamma"]),
     "cosine": (),
     "exp_chi2": frozenset(["gamma"]),
```

Comment:
can you remove the exp_chi2? It's not in ``PAIRWISE_KERNEL_FUNCTIONS`` and I seem to have left it in there by mistake.